main.py

```diff
@@ -144,10 +147,40 @@ def check_date(store, date_to_check):
         '{}: {} -> {}'.format(op, last[astart:aend], booked_list[bstart:bend])
         for op, astart, aend, bstart, bend in  difflib.SequenceMatcher(None, last, booked_list).get_opcodes()
         if op != 'equal'
-    ]
+    ] #Contains the difference between the current details and the previous history file.
 
     if diff:
         pprint(diff)
+        send_email(diff)
+
+def send_email(diff):
+    email_body = str(diff).replace('–','-').replace('\', \'', '\',\n\'') #Changing the dashes is more legible from terminal, can remove that for real emails.
```

Comment:
make a change to the diff itself rather than replace if you really want to change the format